MBBSEmu.Tests/ExportedModules/Majorbbs/random_Tests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class random_Tests : ExportedModuleTestBase
+    {
+        private const int GENRDN_ORDINAL = 315;
+        private const int LNGRND_ORDINAL = 390;
+        private const int RAND_ORDINAL = 486;
+
+        [Theory]
+        [InlineData(2, 2)]
+        [InlineData(1, 3)]
+        [InlineData(5, 20)]
+        [InlineData(2500, 10000)]
+        [InlineData(10000, 65000)]
+        [InlineData(1, 6000)]
+        public void genrdnTest(ushort valueMin, ushort valueMax)
+        {
+            Reset();
+
+            ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, GENRDN_ORDINAL, new List<ushort> { valueMin, valueMax });
+
+            //Verify Results
+            Assert.InRange(mbbsEmuCpuRegisters.AX,valueMin, valueMax);
+        }
+
+        [Fact]
+        public void genrdnTest_invalid()
+        {
+            Reset();
+
+            ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, GENRDN_ORDINAL, new List<ushort> { 10, 2 });
+
+            var expectedValue = 10;
+
+            //Verify Results
+            Assert.Equal(mbbsEmuCpuRegisters.AX, expectedValue);
+        }
+
+        [Theory]
+        [InlineData(1, 65000)]
+        [InlineData(10000, 65000)]
+        [InlineData(10, 20)]
+        [InlineData(11, 12)]
+        [InlineData(32767, 65500)]
```

Comment:
How about adding some values that are actually long? All these test values have high words = 0